app/services/config_report_importer.rb

```diff
@@ -0,0 +1,9 @@
+class ConfigReportImporter < ReportImporter
+  def self.authorized_smart_proxy_features
+    ['Puppet', 'Chef Proxy']
```

Comment:
Original importer had Puppet and Chef feature.
Other importers (features) can inherit from ```ReportImporter``` and add their own feature?